DogTales/dog/dogtales.cpp

```diff
@@ -1,11 +1,12 @@
 #include <dog/dogtales.hpp>
 
 namespace dog {
-DogTales::DogTales(bave::App& app) : bave::Driver(app) {}
+DogTales::DogTales(bave::App& app) : bave::Driver(app), m_player(app, world_space_v) {}
 
 void DogTales::tick() {
 	auto const dt = get_app().get_dt();
 
+	// Update player
```

Comment:
is this kind of comment necessary? `m_player.tick()` is pretty self-explanatory to me

Reply:
Removed. Player cant be initialized.